app/src/main/res/layout/activity_select_verification_method.xml

```diff
@@ -1,48 +1,85 @@
-        android:text="@string/select_visual_verification"
-        android:textSize="@dimen/select_verification_method_button_text_size" />
-
-    <Button
-        android:id="@+id/select_rhythm_verification"
-        android:layout_width="fill_parent"
-        android:layout_height="wrap_content"
-        android:onClick="openVerificationMethod"
-        android:padding="@dimen/select_verification_method_button_padding"
-        android:text="@string/select_rhythm_verification"
-        android:textSize="@dimen/select_verification_method_button_text_size" />
-
-    <LinearLayout
+    <RelativeLayout
+        android:layout_width="match_parent"
         android:layout_height="0dp"
+        android:layout_margin="@dimen/select_verification_method_button_margin"
+        android:layout_weight="1"
+        android:elevation="24dp"
+        tools:targetApi="lollipop">
+
+        <ImageButton
+            android:id="@+id/btn_select_visual_verification"
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:background="@color/blue_btn_bg_pressed_color"
+            android:contentDescription="@string/select_visual_verification"
+            android:hapticFeedbackEnabled="true"
+            android:onClick="onVerificationMethodClick"
+            android:scaleType="centerCrop"
+            app:srcCompat="@drawable/visualverification_farm"
+            tools:targetApi="lollipop" />
+
+        <TextView
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:background="@color/select_verification_method_description_background_color"
+            android:padding="@dimen/select_verification_method_description_padding"
+            android:text="@string/by_tapping_on_an_image"
+            android:textAlignment="center"
+            android:textColor="@color/white" />
+    </RelativeLayout>
+
+    <RelativeLayout
         android:layout_width="match_parent"
+        android:layout_height="0dp"
+        android:layout_marginTop="0dp"
+        android:layout_margin="@dimen/select_verification_method_button_margin"
         android:layout_weight="1"
-        android:padding="@dimen/select_verification_method_checkbox_padding"
-        android:gravity="center|bottom"
-        android:orientation="vertical" >
+        android:elevation="24dp"
+        tools:targetApi="lollipop">
+
+        <ImageButton
+            android:id="@+id/btn_select_rhythm_verification"
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:background="@color/blue_btn_bg_pressed_color"
+            android:contentDescription="@string/select_rhythm_verification"
+            android:hapticFeedbackEnabled="true"
+            android:onClick="onVerificationMethodClick"
+            app:srcCompat="@drawable/rhythm_pairing"
+            tools:targetApi="lollipop" />
```

Comment:
Because we're using some new features. Not sure which one it was in this case, probably hapticFeedbackEnabled